backend/app/Http/Controllers/Api/TestController.php

```diff
@@ -0,0 +1,217 @@
+<?php
+
+namespace App\Http\Controllers\Api;
+
+use App\Http\Controllers\Controller;
+use App\Models\Test;
+use Illuminate\Support\Facades\Validator;
+use Illuminate\Http\Request;
+
+class TestController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $tests = Test::all();
+
+        return response()->json([
```

Comment:
Ich würde an deiner Stelle einen ApiController schreiben, in dem ich den Code auslagere der sich immer wieder wiedrholt. Schau dir dazu mal diesen Blogpost an. https://cerwyn.medium.com/laravel-generalizing-api-response-error-handling-85646a195fea 
Dann könntest du hier deinen Code z.B. so verkürzen.
```suggestion
        return $this->successResponse($tests);
```